app/feature/feature-login/src/main/kotlin/com/hedvig/android/feature/login/genericauth/SsnInputScreen.kt

```diff
@@ -56,9 +56,9 @@ fun SSNInputScreen(
         .verticalScroll(rememberScrollState()),
     ) {
       Spacer(Modifier.height(60.dp))
-      Text(
+      HedvigText(
         text = stringResource(R.string.zignsec_login_screen_title),
-        style = MaterialTheme.typography.headlineMedium,
+        style = HedvigTheme.typography.headlineMedium,
```

Comment:
aren't these two very different in size?

Reply:
28 vs 24 sp, yeah they are different.
These screens are all already getting like absolutely no traffic, I thought it's fine to take such shortcuts tbh